### ISSUE TITLE
Updates for Nextcloud 22/23

### ISSUE THREAD

**Issue by neufeind** [nickv-nextcloud/talk_simple_poll#27]
Can you please consider updating this app for Nextcloud 22/23? Maybe it "just works" by raising the max-version?

**Comment by nickvergessen** [comment]
If so you can use the force enable option in the app management section for the time being

**Comment by sokai** [comment]
FYI: *Forcing* the enabling just works with `v25.0.2` (, but I didn't tested the app = polls).

**Comment by nickvergessen** [comment]
25.0.2 is far behind. You should really update to the latest 25 version.

Other than that, a polls feature got deeper integrated into Talk and can be used out of the box

**Comment by sokai** [comment]
@nickvergessen

> 25.0.2 is far behind. You should really update to the latest 25 version.

You are right and I updated to `25.0.6` some minutes after commenting … B-)

> Other than that, a polls feature got deeper integrated into Talk and can be used out of the box

Sry, I can't get it … :-/ – This app was installed some time ago, when I needed this feature. The last time I used Talk (or this app) was a year ago …
Anyway, as a admin I see this app is not *officially approved* for my current NC version and I think *forcing* enabling shouldn't be the way to use this app.

BR

**Comment by nickvergessen** [comment]
Yeah, this app should not be used at all anymore. Use the native polls? They work in mobile clients and browsers, user's dont have to type answers but can click/tab them, etc.
